agent/util-scripts/pbench-register-tool-trigger

```diff
@@ -19,17 +20,24 @@ stop_trigger=""
 
 function usage() {
 	printf "usage:\n"
-	printf "$script_name --group=<tool group> --start-trigger=<string to match> --stop-trigger=<string to match>"
+	printf "$script_name [--group=<tool group>] --start-trigger=<string to match> --stop-trigger=<string to match>\n"
 }
 
+case $# in
+    0)
+        usage
+        exit 1
+        ;;
+esac
+
 # Process options and arguments
 opts=$(getopt -q -o h --longoptions "help,group:,start-trigger:,stop-trigger:" -n "getopt.sh" -- "$@");
 if [ $? -ne 0 ]; then
 	printf "\n"
 	printf "%s\n" $*
 	printf "$script_name: you specified an invalid option\n\n"
 	usage
-	exit 1
+	exit 2
```

Comment:
It is not clear to me that we want to start incrementing the exit code on the `b0.69` branch.

Reply:
Beyond that, do we really want a different code for each individual error?  It's more conventional to use specific codes for error _types_...so all of the bad and missing option errors would be `2`'s, while operational failures (such as failure to create files or directories) would be `1`'s.

Reply:
Reverted to a single error code (1).